doc/man3/OSSL_CMP_ATAV_set0.pod

```diff
@@ -0,0 +1,83 @@
+=pod
+
+=head1 NAME
+
+OSSL_CMP_ATAV,
+OSSL_CMP_ATAV_create,
+OSSL_CMP_ATAV_set0,
+OSSL_CMP_ATAV_get0_type,
+OSSL_CMP_ATAV_get0_value,
+OSSL_CMP_ATAV_push0_stack_item
+- OSSL_CMP_ATAV utility functions
+
+=head1 SYNOPSIS
+
+  #include <openssl/cmp.h>
+
+  typedef OSSL_CRMF_ATTRIBUTETYPEANDVALUE OSSL_CMP_ATAV;
+  OSSL_CMP_ATAV *OSSL_CMP_ATAV_create(ASN1_OBJECT *type, ASN1_TYPE *value);
+  void OSSL_CMP_ATAV_set0(OSSL_CMP_ATAV *atav, ASN1_OBJECT *type,
+                          ASN1_TYPE *value);
+  ASN1_OBJECT *OSSL_CMP_ATAV_get0_type(const OSSL_CMP_ATAV *atav);
+  ASN1_TYPE *OSSL_CMP_ATAV_get0_value(const OSSL_CMP_ATAV *atav);
+
+  int OSSL_CMP_ATAV_push0_stack_item(STACK_OF(OSSL_CMP_ATAV) **sk_p,
+                                     OSSL_CMP_ATAV *atav);
+
+=head1 DESCRIPTION
+
+B<OSSL_CMP_ATAV> is a short hand of B<OSSL_CRMF_ATTRIBUTETYPEANDVALUE>,
+defined in RFC 4211 Appendix B.
+It is typically used in CertRequest structures,
+but also in CertReqTemplateContent structures for key specifications.
+
+OSSL_CMP_ATAV_create() creates a new B<OSSL_CMP_ATAV> structure and fills it in.
+It combines OSSL_CMP_ATAV_new() and OSSL_CMP_ATAV_set0().
+
+OSSL_CMP_ATAV_set0() sets the I<atav> with an infoType of I<type> and an
+infoValue of I<value>. This function uses the pointers I<type> and I<value>
+internally, so they must B<not> be freed up after the call.
+
+OSSL_CMP_ATAV_get0_type() returns a direct pointer to the infoType in the
+I<atav>.
+
+OSSL_CMP_ATAV_get0_value() returns a direct pointer to the infoValue in
+the I<atav> as generic B<ASN1_TYPE> pointer.
+
+OSSL_CMP_ATAV_push0_stack_item() pushes I<atav> to the stack pointed to
+by I<*sk_p>. It creates a new stack if I<*sk_p> points to NULL.
+
+=head1 NOTES
+
+CMP is defined in RFC 4210. CRMF is defined in RFC 4211.
+
+=head1 RETURN VALUES
+
+OSSL_CMP_ATAV_create() returns a pointer to the ATAV structure on success,
+or NULL on error.
+
+OSSL_CMP_ATAV_set0() does not return a value.
+
+OSSL_CMP_ATAV_get0_type() and OSSL_CMP_ATAV_get0_value()
+return the respective pointer or NULL if their input is NULL.
+
+OSSL_CMP_ATAV_push0_stack_item() returns 1 on success, 0 on error.
+
+=head1 SEE ALSO
+
+L<OSSL_CMP_ITAV_new0_certReqTemplate(3)>, L<ASN1_TYPE_set(3)>
+
+=head1 HISTORY
+
+The OpenSSL CMP support was added in OpenSSL 3.0.
```

Comment:
This should document when these functions were added (3.1)

Reply:
Oops, this was a copy&paste mistake, fixed.